### PR TITLE
`treehouses services librespeed` (fixes #1217)

### LINKED ISSUE #1217: add service librespeed
https://hub.docker.com/r/linuxserver/librespeed

### PR DESCRIPTION
**Pull request by LordJashin32**
tests all worked and
```bash
root@treehouses:~/cli# ./cli.sh services librespeed install
Pulling librespeed ... digest: sha256:3b5f9d6c0e1ef70c1d..Pulling librespeed ... status: downloaded newer image fo..Pulling librespeed ... done
librespeed installed
modify default environment variables by running 'cli.sh services librespeed config edit'
root@treehouses:~/cli# ./cli.sh services librespeed up
valid yml
Creating network "librespeed_default" with the default driver
Creating librespeed_librespeed_1 ... done
librespeed built and started
root@treehouses:~/cli# ./cli.sh services librespeed url
192.168.0.21:80

```

### CODE REVIEW

**Comment by kjong** [comment]
The tests have incorrect directories. Remove the `../` in `run` for each test.

File: tests/services/librespeed.bats [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bats
+load ../test-helper
+
+@test "$clinom services librespeed info" {
+  run "../${clicmd}" services librespeed info
```
Comment (kjong) [inline tests/services/librespeed.bats:5]: Need to fix the directories.

Remove `../` in `run`.

File: services/install-librespeed.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    echo "      - PGID=\${PGID}"
+    echo "      - TZ=\${TZ}"
+    echo "      - PASSWORD=\${PASSWORD}"
+    echo "      - DB_TYPE=\${DB_TYPE}"
+    echo "      - DB_NAME=\${DB_NAME}"
+    echo "      - DB_HOSTNAME=\${DB_HOSTNAME}"
+    echo "      - DB_USERNAME=\${DB_USERNAME}"
+    echo "      - DB_PASSWORD=\${DB_PASSWORD}"
+    echo "    ports:"
+    echo "      - 80:80"
```
Comment (kjong) [inline services/install-librespeed.sh:24]: Conflicting ports with planet

**Comment by kjong** [comment]
Just curious, what does a speed test app need a database for?

File: tests/services/librespeed.bats (hunk 1 above)
Comment (LordJashin32) [reply tests/services/librespeed.bats]: my bad. fixed

**Pull request merged**